WonSeungHyun/.env.sample

```
@@ -0,0 +1,9 @@
+PORT = YOUR PORT NUMBER SHOULD BE HERE!
+
+TYPEORM_CONNECTION = RDBMS_TYPE
+TYPEORM_HOST = 127.0.0.1
+TYPEORM_USERNAME = USERNAME
+TYPEORM_PASSWORD = myPassword
+TYPEORM_DATABASE = myDB
+TYPEORM_PORT = 3306
+TYPEORM_LOGGING =TRUE
```

안녕하세요 승현님 !! 세팅을 너무 잘 해주셨네요 ! 동일하게 HOST, PORT도 수정해 주시면 감사하겠습니다!
```suggestion
PORT = YOUR PORT NUMBER SHOULD BE HERE!

TYPEORM_CONNECTION = RDBMS_TYPE
TYPEORM_HOST = YOUR HOST
TYPEORM_USERNAME = USERNAME
TYPEORM_PASSWORD = myPassword
TYPEORM_DATABASE = myDB
TYPEORM_PORT = YOUR PORT
TYPEORM_LOGGING =TRUE
```